final/src/main/java/ua/epam/spring/hometask/converter/impl/AirDateConverterImpl.java

```diff
@@ -19,38 +14,29 @@
     private AuditoriumConverter auditoriumConverter;
 
     @Autowired
-    private EventConverter eventConverter;
+    private EventConverterImpl eventConverter;
```

Comment:
It's bad practice to work against implementations. Use Converter interface instead. Inject dependencies using bean names in @Qualifier annotation (works together with @Autowired) or any other annotation (@Resource or @Inject).